client/enable-power-assert.js

```diff
@@ -0,0 +1,3 @@
+require('espower-loader')({
+  pattern: 'client/**/*.js',
```

Comment:
mmm
I dont need now because I dont write jsx for test.
If there is someone who wants to write jsx, then`.jsx` will be added .